contracts/WithVestingContract.sol

```diff
@@ -26,15 +27,9 @@ contract WithVestingContract is WeightedTokenDistributor {
     address public vestingContract;
     vestingContractVersion public targetVersion;
 
-    bytes4[2] public vestingReleaseCalls = [
-        bytes4(keccak256("release()")),
-        bytes4(keccak256("release(address)"))
-    ];
-
     constructor ( address _targetToken, uint256 _totalStakeHolders, address[] _stakeHolders, uint256[] _weights) public
     WeightedTokenDistributor(_targetToken, _totalStakeHolders, _stakeHolders, _weights)
-    {
-    }
+    {}

```

Comment:
Empty constructor?

Reply:
Yea, intentional.
Needed to instantiate the `WeightedTokenDistributor`